src/models/contracts.ts

```diff
@@ -111,3 +115,15 @@ export namespace SaveResultsAsExcelRequest {
     >("query/saveExcel");
 }
 // --------------------------------- </ Save Results as Excel Request > ------------------------------------------
+
+// --------------------------------- < Save Results as INSERT Request > ------------------------------------------
+// save results in INSERT format
+export namespace SaveResultsAsInsertRequest {
+    export const type = new RequestType<
+        SaveResultsAsInsertRequestParams,
+        SaveResultRequestResult,
+        void,
+        void
+    >("query/saveInsert");
```

Comment:
Do you need to issue an STS PR to get this endpoint in too?

Reply:
Yup: https://github.com/microsoft/sqltoolsservice/pull/2519